server/app/services/grid_scheduler.rb

```diff
@@ -66,18 +65,6 @@ def should_reschedule_service?(service)
 
   # @param [GridService] service
   # @return [Boolean]
-  def pending_deploys?(service)
-    service.grid_service_deploys.where(started_at: nil).count > 0
-  end
-
-  # @param [GridService] service
-  # @return [Boolean]
-  def active_deploys?(service)
-    service.grid_service_deploys.where(:started_at.gt => 30.minutes.ago, :deploy_state.in => [:created, :ongoing]).count > 0
```

Comment:
This seems mostly equivalent to what `GridSevice.deploying?` now does, except that `:finished_at => nil` is now required for the case of aborted deploys for non-running services.